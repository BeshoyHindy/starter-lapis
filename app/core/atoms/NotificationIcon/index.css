@import 'variables';

@svg-load icon url(./icon.svg) {
    fill: #000;
}
@svg-load icon-hover url(./icon.svg) {
    fill: var(--primary-color);
}


.icon {
  display: inline-block;
  background-image: svg-inline(icon);
  height: 30px;
  width: 30px;
  background-repeat: no-repeat;
  transition: background-image 0.35s ease-in-out;
  &:hover {
    background-image: svg-inline(icon-hover);
  }
}
