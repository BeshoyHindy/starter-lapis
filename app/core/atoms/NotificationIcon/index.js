import React from 'react';
import './index.css';

const NotificationIcon = () => <span styleName="icon" />;

export default NotificationIcon;
