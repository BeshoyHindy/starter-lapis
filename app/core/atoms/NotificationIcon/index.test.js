import React from 'react';
import renderer from 'react-test-renderer';
import NotificationIcon from './index.js';

test('NotificationIcon', () => {
  const component = renderer.create(<NotificationIcon />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
