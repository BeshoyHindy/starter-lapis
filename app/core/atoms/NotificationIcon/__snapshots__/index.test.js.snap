exports[`test NotificationIcon 1`] = `
<span
  styleName="icon" />
`;
